lib/pure/os.nim

```diff
@@ -1668,9 +1684,21 @@ proc copyFile*(source, dest: string) {.rtl, extern: "nos$1",
       if copyFileW(s, d, 0'i32) == 0'i32: raiseOSError(osLastError(), $(source, dest))
     else:
       if copyFileA(source, dest, 0'i32) == 0'i32: raiseOSError(osLastError(), $(source, dest))
+  elif hasCCopyfile:
+    let state = copyfile_state_alloc()
+    # xxx `COPYFILE_STAT` could be used for one-shot `copyFileWithPermissions`.
+    let status = c_copyfile(source.cstring, dest.cstring, state, COPYFILE_DATA)
+    if status != 0:
+      let err = osLastError()
+      discard copyfile_state_free(state)
+      raiseOSError(err, $(source, dest))
+    let status2 = copyfile_state_free(state)
+    if status2 != 0: raiseOSError(osLastError(), $(source, dest))
   else:
     # generic version of copyFile which works for any platform:
-    const bufSize = 8000 # better for memory manager
+    const bufSize = 8192
```

Comment:
What changed? 8192 is still a bad number for the memory manager..

Reply:
@Araq I've now reverted the `bufSize` change but please add a comment in https://github.com/nim-lang/RFCs/issues/330 for proposal 4 explaning the rationale behind 8000, ideally also with a link to some reference, I don't understand and in my experiments, 8000 didn't help (and most/all implementations of copyFile I've seen used a power of 2)